index.html

```diff
@@ -19,51 +16,259 @@
     <title>portfolio</title>
   </head>
   <body>
-    <header id="menu-mobile">
```

Comment:
> * [ ]  One of the requirements says that when building the desktop version your mobile version should remain the same But in your design I have found out extra space at the right of your design kindly fix it. check here point_down

- [ ] As required by the previous reviewer, your website presents some overflow, try to give` overflow: hidden;` to those sections that presents the issue

![pin2](https://user-images.githubusercontent.com/57562869/150498786-de510d1e-186f-4005-b7b0-cce80c3d845c.png)

Reply:
Thank you I will implement the first change but please, I fixed the issue that you have raised on the second point, please consider viewing my mobile version on a mobile screen width, which was 375px in figma. Thank you.

Reply:
I can't add overflow hidden to my html tag because that would disable scrolling altogether. What I did do was to place overflow-x: hidden, to enable sideways scrolling thank you.

Reply:
Try to give the `overflow: hidden` property inside `boby` then define the  `html `tag and give `overflow: scroll` property to make scrolling your website,